js/src/Map.js

```diff
@@ -208,15 +208,42 @@ export class LeafletMapView extends utils.LeafletDOMWidgetView {
     }).then(view => {
       this.obj.addControl(view.obj);
 
+
       // Trigger the displayed event of the child view.
       this.displayed.then(() => {
         view.trigger('displayed', this);
       });
-
       return view;
     });
   }
 
+  remove_subitem_view(child_view) {
+    if(child_view['name'].includes('Control')){
```

Comment:
I agree with the fact that this way of proceeding is a bit week. Since for the moment, in ipyleaflet all control includes the 'Control' subname, this should work but we should indeed think of a better way. Maybe have uniques Add/Remove methods working both for layers and controls ?

Reply:
Finally we have updated `add_subitem_model` and `remove_subitem_view` using `instanceof`checks. This is in done in [e514e31](https://github.com/jupyter-widgets/ipyleaflet/pull/1011/commits/e514e31cd34dfb71f55fda8a46bbdab2b110ae60)